packages/ui-options/src/Options/index.tsx

```diff
@@ -183,7 +181,8 @@ class Options extends Component<Props> {
           margin="none"
           padding="none"
           background="primary"
-          aria-labelledby={renderLabel && this._labelId}
+          // TODO renderLabel is returned is renderLabel is a function, this is likely a bug
+          aria-labelledby={(renderLabel as string) && this._labelId}
```

Comment:
Likely a bug.. ir renderLabel is a function its returned. I think it should return _labelId if renderLabel is not undefined.